plasma_framework/contracts/src/framework/utils/ExitPriority.sol

```diff
@@ -6,32 +6,42 @@ library ExitPriority {
 
     using PosLib for PosLib.Position;
 
+    uint256 constant private SIZEOF_TIMESTAMP = 32;
+    uint256 constant private SIZEOF_TXPOS = 54;
+    uint256 constant private SIZEOF_EXITID = 168;
+
     /**
      * @dev Returns an exit priority for a given UTXO position and a unique ID.
      * The priority for Plasma M(ore)VP protocol is a combination of 'exitableAt' and 'txPos'.
-     * Since 'exitableAt' only provides granularity of block, add 'txPos' to provide priority for a transaction.
+     * Since 'exitableAt' only provides granularity of block, adding 'txPos' gives higher priority to transactions added earlier in the same block.
      * @notice Detailed explanation on field lengths can be found at https://github.com/omisego/plasma-contracts/pull/303#discussion_r328850572
      * @param exitId Unique exit identifier
      * @return An exit priority
      *   Anatomy of returned value, most significant bits first
-     *   42 bits  - timestamp in seconds (exitable_at); we can represent dates until year 141431
```

Comment:
Creating the sum of bits over timestamp (42) and blocknum (54) and exit id (160) before was 256. With the new scheme it's 254?

Reply:
Dammit, you're right 🤦 
txpos should be 56 instead of 54